README.md

```diff
@@ -46,19 +46,26 @@ You have multiple options:
 
 - Basic building tools
 - CMake
-- Qt5, QtWebEngine and Kf5Notifications
+- Qt5 and QtWebEngine
 - **PipeWire** (it currently doesn't work with PulseAudio)
 - Git
+- notification-daemon
```

Comment:
Is this a dependency which has to be installed?

Reply:
> Is this a dependency which has to be installed?

No, this is the basic notification daemon. KDE is usually bloated and was a pain to remove knotifications from my system.
What about making a installation script to identify the DEs?

Reply:
Various [debian packages](https://packages.debian.org/buster/notification-daemon), for example, use notification-daemon as part of the recipe.